tensorflow_datasets/core/dataset_builder_beam_test.py

```diff
@@ -66,7 +75,7 @@ def _split_generators(self, dl_manager):
         ),
     ]
 
-  def _compute_metadata(self, examples, num_examples):
+  def _compute_metadata(self, examples: Iterable, num_examples: int) -> None:
```

Comment:
Please use specific typing `Iterable[...]` rather than generic `Iterable`

Reply:
```Invalid type annotation 'Ellipsis'  [invalid-annotation]```

Reply:
I meant you should specif a type, like `Iterable[str]`, `Iterable[Any]`...

Reply:
Done